docs/_sass/uswds/src/settings/_settings-general.scss

```diff
@@ -39,7 +39,7 @@ Show updates and notifications.
 */
 
 $theme-show-compile-warnings: true !default;
-$theme-show-notifications: true !default;
+$theme-show-notifications: false !default;
```

Comment:
Ensures the build output is less verbose.

With it enabled, the output generated a WARNING to show release notes:

`````
% bundle exec jekyll serve      
Configuration file: /Users/swells/Documents/src/github/CrowdStrike/community/docs/_config.yml
            Source: /Users/swells/Documents/src/github/CrowdStrike/community/docs
       Destination: /Users/swells/Documents/src/github/CrowdStrike/community/docs/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
WARNING: 

--------------------------------------------------------------------
✉ USWDS Notifications
--------------------------------------------------------------------
2.10.1:
- We updated the `$theme-breadcrumb-background-color` default
  to `default` from `'white'`. 
- We updated the `$theme-alert-icon-size` default to `5` from `4`.
- [Footer] We updated `usa-footer__logo-heading` to use a `p`.
  instead of an `h3`. 
--------------------------------------------------------------------
2.10.0:
- We updated the Spanish language link to USAGov in the identifier. 
  The link should now read 'Visite USAGov en Español'. 
- We improved the accessibility of SVG images in the banner. 
  See the full release notes for more information.
--------------------------------------------------------------------
2.9.0:
- We updated the formatting of `usa-legend`. If you use a visible 
  legend and wish to preserve the old styling, add a 
  `usa-legend--large` modifier to the element.
--------------------------------------------------------------------
2.8.0:
- We updated the text of the gov banner and its settings.
  Please see the full release notes for more information and update
  your project's banner as soon as possible.
--------------------------------------------------------------------
2.7.1:
- We added `not dead` to our autoprefixer settings and now use a 
  `.browserslistrc` file for these options.
--------------------------------------------------------------------
`````

Now it doesn't:
`````
% bundle exec jekyll serve                                
Configuration file: /Users/swells/Documents/src/github/CrowdStrike/community/docs/_config.yml
            Source: /Users/swells/Documents/src/github/CrowdStrike/community/docs
       Destination: /Users/swells/Documents/src/github/CrowdStrike/community/docs/_site
 Incremental build: disabled. Enable with --incremental
      Generating... 
                    done in 3.9 seconds.
 Auto-regeneration: enabled for '/Users/swells/Documents/src/github/CrowdStrike/community/docs'
    Server address: http://127.0.0.1:4000
  Server running... press ctrl-c to stop.
`````